docs/how-to/move-ipfs-installation.md

```diff
@@ -24,7 +24,7 @@ The easiest way to move your IPFS installation is to grab the `.ipfs` folder, an
 1. Make a copy of the `.ipfs` directory:
 
    ```bash
-   cp --recursive --verbose .ipfs ipfs-backup
+   cp --recursive --verbose $([[ -z $(cp --help | grep "\-\-reflink" | head -n1) ]] || echo -n "--reflink=auto") .ipfs ipfs-backup
```

Comment:
This looks good, but some users might not understand what's happening here:

> `[...] $([[ -z $(cp --help | grep "\-\-reflink" | head -n1) ]] || echo -n "--reflink=auto") [...]`

Can you please add an explanation as to what this section is doing. Your comments in this PR should work as an explanation.

Reply:
Done